src/components/atoms/LogoType/LogoType.tsx

```diff
@@ -0,0 +1,27 @@
+import styles from './LogoType.module.css'
+import Image from 'next/image'
+import AmberDefault from '@/../public/logotypes/amber-default.svg'
+import AmberChristmas from '@/../public/logotypes/amber-christmas.svg'
+
+export default function SeasonalLogoType() {
+    const month = new Date().getMonth();
+    if (month === 11 || month === 0) {
+        return <Image
+            src={AmberChristmas}
+            alt="Amber logo christmas edition"
+            style={{
+                position: 'relative',
+                width: 'auto',
+                // This image is 150% of the default logo
+                height: '150%',
+                top: '-50%',
+                transform: 'translateX(-4%)',
```

Comment:
I calculated that hat takes 4% of the space from the left side

<img width="371" alt="Screenshot 2024-12-16 at 16 21 21" src="https://github.com/user-attachments/assets/8899e708-5c66-4070-9242-e75ec19f2220" />